src/index.js

```diff
@@ -0,0 +1,167 @@
+// 'use strict';
+import 'regenerator-runtime/runtime';
+import axios from 'axios';
+
+const state = {
+  tempCount: 60,
+  cityName: 'Seattle',
+  latitude: 0,
+  longitude: 0,
+};
+
+function convertTempKtoF(tempInK) {
+  return Math.round(1.8 * (tempInK - 273) + 32);
+}
+
+const getCityCoordinates = () => {
+  axios
+    .get('https://fast-dawn-95769.herokuapp.com/location', {
+      params: {
+        q: state.cityName,
+      },
+    })
+    .then((response) => {
+      state.latitude = response.data[0].lat;
+      state.longitude = response.data[0].lon;
+      console.log('Success: ', response.data);
+
+      const getCityTemp = () => {
```

Comment:
This works nicely! Also consider having this as a function of its own outside of the `.getCityCoordinates` function for more streamlined readability. And only have the function call it in the `.then()` callback.